code/__DEFINES/~skyrat_defines/DNA.dm

```diff
@@ -62,8 +62,17 @@
 #define BODY_SIZE_MAX			1.25
 #define BODY_SIZE_MIN			0.85
 
-#define MANDATORY_FEATURE_LIST list("mcolor" = "FFB","mcolor2" = "FFB","mcolor3" = "FFB","ethcolor" = "FCC","skin_color" = "FED","flavor_text" = "","breasts_size" = 1,"breasts_lactation" = FALSE,"penis_size" = 13,"penis_girth" = 9,"penis_taur_mode" = TRUE,"balls_size" = 1, "body_size" = BODY_SIZE_NORMAL)
+#define SHEATH_NONE			"none"
+#define SHEATH_NORMAL		"sheath"
+#define SHEATH_SLIT			"slit"
```

Comment:
```suggestion
#define SHEATH_NONE			"None"
#define SHEATH_NORMAL		"Sheath"
#define SHEATH_SLIT			"Slit"
```

Reply:
They're used as text to compile the string to show a icon state, that's why they're lowercase.. hm..

Reply:
Can you just use lowercase when getting the icon state? It looks a bit tacky having everything else as capitalized and then randomly the sheath bit completely lowercase imo

Reply:
True, I'mma do that